tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2555,7 +2597,7 @@ public function testMergeManyFieldList()
             ['id' => 1, 'comment' => 'Changed 1', 'user_id' => 20]
         ];
         $marshall = new Marshaller($this->comments);
-        $result = $marshall->mergeMany($entities, $data, ['fieldList' => ['id', 'comment']]);
+        $result = $marshall->mergeMany($entities, $data, [$fields => ['id', 'comment']]);
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?